neubot/config.py

```diff
@@ -34,6 +34,7 @@
     "bittorrent.piece_len": 1 << 17,
     "bittorrent.port": 6881,
     "bittorrent.watchdog": 300,
+    "bittorrent.max_message": , #XXX
```

Comment:
In [neubot/bittorrent/config.py](https://github.com/neubot/neubot/blob/master/neubot/bittorrent/config.py#L43) was `1 << 18` and we can use the same here